.github/workflows/lint-react.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: "Lint react"
+
+on:
+  pull_request:
+    paths-ignore:
+      - '*.md'
+      - 'docs/*'
```

Comment:
Yeap, perfectly right. I can switch this to instead only run for `webui/*` paths if you think that's better. I wasn't sure of any tooling dependencies or such that go across top-level directories, so I went with this approach.

Reply:
`/webui/*` dir should cover everything except if the node version changes which is captured in this file?. (or if we move the webui dir ofc)

ahh thinking more about it we also depend on `/proto` dir since that generates our api bindings. and that depends on `/bindings` since that defines what the generator is... ughh maybe excluding some paths is easier. let's leave it as is to be safe